src/main.rs

```diff
@@ -77,6 +77,32 @@ fn main_inner() -> Result<(), String> {
     setup_tracing(&args);
     trace!(?args, "commandline arguments");
 
+    let subargs = args.subcommand().1.unwrap_or(exit(1));
```

Comment:
Check: Unreachable call

Reply:
Ah yes. `exit(1)` is NOT lazily evaluated, so the program stops here.